vitest.config.ts

```diff
@@ -89,17 +90,33 @@ export default defineConfig({
         },
       },
       {
-        extends: true,
+        extends: './packages-private/vapor-e2e-test/vite.config.ts',
```

Comment:
The test config reuses the Vites config from a private repo instead of the main one (so, aliases and defines are not actually inherited, but tests seem to run ok...)